napari/_vispy/overlays/bounding_box.py

```diff
@@ -26,6 +26,16 @@ def _on_bounds_change(self):
         bounds = self.layer._display_bounding_box_augmented_data_level(
             self.layer._slice_input.displayed
         )
+        if (
+            len(self.layer._slice_input.displayed) == 3
+            and self.layer.multiscale
+        ):
+            # for 3D multiscale layers, the lowest data level is displayed
+            # and we need the augmented bounding box
+            bounds = self.layer._display_bounding_box_at_level(
+                self.layer._slice_input.displayed, len(self.layer.data) - 1
+            ) + np.array([[-0.5, 0.5]])
```

Comment:
Could you add a comment about the `-0.5, 0.5` part?

Reply:
That's the "augmented" part, to match the existing code.

Reply:
`_display_bounding_box_augmented_data_level` should work and then we wouldnt need this PR -- it's what the code uses -- but it ends up using the data_level from the 2D view when the overlay is updated. I did some testing in the issue, at some point after display = 3 `_display_bounding_box_augmented_data_level` is updated and correct, but not when the overlay is updated.

`_display_bounding_box_augmented` doesn't allow you to specify the `data_level`.

So the easiest thing was to use the method that does allow one to specify the level and then augment it manually.

Reply:
Why can we not set the data level to the lowest resolution on switch into 3d?

Reply:
That's what I do here? `len(self.layer.data) - 1` is the lowest level.
Unless I'm missing something.

Reply:
We shouldn't need data level, I think @brisvag is saying that layer._display_bounding_box_augmented is in world coordinates so should work regardless? We know the layer extents in world coordinates which would make data levels moot.

Reply:
using `layer._display_bounding_box_augmented` returns the data_level 0 bounding box regardless of 2d/3d, zoom, etc. so that option is always wrong 😞

Reply:
But that should be correct *in 3d*, no? Or at least, it should... I'm testing stuff manually a bit and none of this behaves like I expect :cry:

Reply:
No, in 3D multiscale uses the lowest pyramid level -- 0 is the highest.
Currently, the overlay uses the level currently displayed, which actually does work some of the time -- like when you are zoomed out so that the lowest level is displayed.